examples/align/align_and_minimize_marker_velocity.py

```diff
@@ -98,6 +98,7 @@ def prepare_ocp(
         U_bounds,
         objective_functions,
         ode_solver=ode_solver,
+        number_of_finite_elements=10,
```

Comment:
That doesn't change the behavior of the test??

Reply:
I know it's weird, but it the final values don't change ?!

Reply:
I sorted this out, it is actually just because 5 and 10 are actually equivalent since 5 is enough. If we small it down to 1, it fails, so this is okay!